### PR TITLE
Skip Version Verification and Update in BOM Releases

### PR DESCRIPTION
**Pull request by alzimmermsft**
Updates the BOM archetype YML to skip version verification and update during the release pipeline. This is done as YMLs are released on a different cadence and won't be using the same versions listed in the versioning scripts.

### CODE REVIEW

File: eng/pipelines/templates/stages/archetype-sdk-bom.yml [hunk 1]
```diff
@@ -2,6 +2,9 @@ parameters:
   Artifacts: []
   SDKType: bom
   ServiceDirectory: not-specified
+  Skip:
+    UpdatePackageVersion: true
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/archetype-sdk-bom.yml]: This should be:
```
Skip.UpdatePackageVersion: true
Skip.VersionVerification: true
```

Comment (alzimmermsft) [reply eng/pipelines/templates/stages/archetype-sdk-bom.yml]: I was wondering what the syntax on this was 😃

Comment (alzimmermsft) [reply eng/pipelines/templates/stages/archetype-sdk-bom.yml]: Updated to use `Skip.UpdatePackageVersion` and `Skip.VersionVerification`.

**Pull request merged**